common/security.py

```
@@ -0,0 +1,55 @@
+    )
+    return encoded_jwt
+
+def decode_access_token(token: str) -> str | None:
+    try:
+        payload = jwt.decode(token,
+                             settings.SECRET_KEY,
+                             algorithms=[settings.ALGORITHM]
+                             )
+        email: str= payload.get("sub")
+        return email
+    except JWTError as e:
```

Ruff (0.14.11)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)